spacy_llm/tests/tasks/misc/el_kb_data.yml

```diff
@@ -2,48 +2,63 @@ entities:
   Q100:
       name: "Boston"
       desc: "city in and state capital of Massachusetts, United States"
+      embedding: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
What are these embeddings used for, in the context of the LLM EL component?

Reply:
For nothing as of now. We could drop them from the file and generate the KB with arbitrary embeddings. I thought it'd be nice to have the possibility of having a properly initialized KB for later usage, but we could make the `embeddings` field optional?

Reply:
Agreed: it's nice to provide the feature to use in the context of spaCy's entity linker, but for spacy-llm we don't need it, so I wouldn't confuse the example use-case with it.

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/bb54f3da0f66339acce4e0d185932cf560d71f07 and https://github.com/explosion/spacy-llm/pull/241/commits/6c2277affa658b9b586fe4d80f904f2825d8e00b.